ReflectionAnalyzers/ReflectionAnalyzers.csproj

```diff
@@ -20,18 +24,25 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>0.1.20.0-dev</Version>
+    <Version>0.1.20-dev</Version>
+    <DevelopmentDependency>true</DevelopmentDependency>
     <Description>Analyzers for System.Reflection</Description>
-    <PackageLicenseUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</PackageLicenseUrl>
-    <RepositoryUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</RepositoryUrl>
-    <PackageTags>Reflection Roslyn Diagnostic Analyzer DotnetAnalyzers</PackageTags>
+    <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</PackageProjectUrl>
     <PackageReleaseNotes>BUGFIX REFL031: conditional expression.
 BUGFIX REFL031: if statement.</PackageReleaseNotes>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
   </PropertyGroup>
 
-  <Target Name="PostBuild" AfterTargets="PostBuildEvent" Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <Exec Command=".paket\paket.exe pack publish" WorkingDirectory="$(SolutionDir)" />
-  </Target>
+  <ItemGroup>
+    <None Update="tools\*" Pack="true" PackagePath="" />
+    <None
+      Include="$(OutputPath)\$(AssemblyName).dll;$(OutputPath)\Gu.Roslyn.Extensions.dll"
+      Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
See the .NET Compiler Platform SDK's analyzer project template.

Reply:
Are the ps1 files added elswewhere?

Reply:
Yes, that's line 38:

```xml
<None Update="tools\*" Pack="true" PackagePath="" />
```